### PR TITLE
enable v1 in hack/test-integration.go

### PR DESCRIPTION
**Pull request by caesarxuchao**
enable v1 integration tests.

The first 2 commits are in #9101.
#9070 #7018

@bgrant0607 @krousey @nikhiljindal

### CODE REVIEW

File: pkg/api/latest/latest_test.go [hunk 1]
```diff
@@ -21,6 +21,7 @@ import (
 	"testing"
 
 	internal "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	_ "github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1"
```
Comment (bgrant0607) [inline pkg/api/latest/latest_test.go]: Why is this needed, while v1beta3 is not?

Comment (caesarxuchao) [inline pkg/api/latest/latest_test.go]: Looks like it's not needed. Maybe it's an accident when I do rebase.

**Comment by bgrant0607** [comment]
LGTM
